sky/jobs/utils.py

```diff
@@ -352,29 +442,31 @@ def stream_logs_by_id(job_id: int, follow: bool = True) -> str:
             # the next round of status check.
             if (handle is None or managed_job_status !=
                     managed_job_state.ManagedJobStatus.RUNNING):
-                status_str = ''
-                if (managed_job_status is not None and managed_job_status !=
-                        managed_job_state.ManagedJobStatus.RUNNING):
-                    status_str = f' (status: {managed_job_status.value})'
+                status_str = (
+                    f' (status: {managed_job_status.value})'
+                    if managed_job_status is not None and managed_job_status !=
+                    managed_job_state.ManagedJobStatus.RUNNING else '')
```

Comment:
same, to me the if-else oneliner only looks neat if it is concise. And as long as it gets lengthy it is harder to read. Can we revert it?